geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -3,11 +3,11 @@
     {{ annotation_config|json_script:"annotation-config" }}
 {% elif perms.corpus.change_document and document.iiif_images %}
     {# NOTE: document.iiif_images check is temporary, remove once we can use generated canvas ids #}
-    <a class="editor-navigation" href="{% url 'corpus:document-transcribe' document.id %}" data-turbo="false">
+    <a class="editor-navigation" href="{% url 'corpus:document-add-transcription' document.id %}" data-turbo="false">
         <i class="ph-plus-circle"></i>
         <span>Add a new transcription</span>
     </a>
-    {% for ed in document.digital_editions.all %}
+    {% for ed in document.editions.all %}
```

Comment:
If a footnote was added to this document without adding any digital edition content, i.e. adding a "new transcription" through this new view without actually transcribing lines, then it wouldn't show up in `digital_editions`, and thus wouldn't appear in the list. Perhaps that means we need to adjust the digital editions logic, though I thought it might also make sense to allow admins to add digital versions here for editions that are not yet digital.

This also makes me wonder if we should have a uniqueness check on Footnote for source + content object pairs; I was able to add duplicate footnotes by accident (it didn't show up in the `digital_editions` list here so I thought it wasn't created).

Reply:
No longer needed with the new logic (no Footnotes are created).